### PR TITLE
Adding new lines

### PR DESCRIPTION
**Pull request by howlowck**
In markdown, two spaces at the end of the line = linebreak

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-echo "# codereviewtest" >> README.md
-git init
-git add README.md
```
Comment (saroup) [inline README.md:3]: I don't think this is correct

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
-git commit -m "first commit"
-git remote add origin git@github.com:saroup/codereviewtest.git
-git push -u origin master
\ No newline at end of file
+echo "# codereviewtest" >> README.md
```
Comment (howlowck) [inline README.md:1]: this example

File: README.md (hunk 1 above)
Comment (howlowck) [reply README.md:3]: interesting...

Comment (saroup) [reply README.md:3]: this is another test.. adding a comment would this start a new review?

**Pull request closed without merging**